docs/technologies.md

```diff
@@ -0,0 +1,423 @@
+# Technologies
+
+## Overview
+
+BigFlow provides support for the main big data data processing technologies on GCP:
+
+* [Dataflow](https://cloud.google.com/dataflow) (Apache Beam)
+* [BigQuery](https://cloud.google.com/bigquery)
+* [Dataproc](https://cloud.google.com/dataproc) (Apache Spark)
+
+However, **you are not limited** to these technologies. The only limitation is the Python language. What is more, you can
+mix all technologies in a single workflow.
+
+The provided utils allow you to build workflows easier and solve problems that must have been solved anyway.
+
+The BigFlow [project starter](./scaffold.md) provides an example for each technology. Before you dive into the next chapters, 
+[create an example project](./scaffold.md#start-project) using the starter.
+
+## Dataflow (Apache Beam)
+
+A BigFlow project is a Python package. Apache Beam supports [running jobs as a Python package](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#multiple-file-dependencies).
```

Comment:
**The** BigFlow project

Reply:
The standard BigFlow project is a Python package